doc/tutorial/sequence_file/sequence_file_solution4.cpp

```diff
@@ -58,7 +58,7 @@ int main()
         return std::ranges::size(get<field::SEQ>(rec)) >= 5;
     });
 
-    for (auto & rec : fin | length_filter)
+    for (auto & rec : fin | length_filter | std::view::take(2))
```

Comment:
remove this change. The exercise says **all** records.
Same for the others below